config/metadata/configMeta.yaml

```diff
@@ -1405,15 +1440,16 @@ groups:
         valuetype: nondefault
         default: 3s
         reload: true
-        summary: is the minimum time that stress relief will stay enabled.
-        description: >
-          Used when switching into Monitor mode. When stress monitoring is
-          enabled, it will start up in stressed mode for a at least this amount
-          of time to try to make sure that Refinery can handle the load before
-          it begins processing it in earnest. This is to help address the
-          problem of trying to bring a new node into an already-overloaded
-          cluster. If this duration is 0, Refinery will not start in stressed
-          mode, which will provide faster startup at the possible cost of
+        summary: is the minimum time that Stress Relief will stay enabled.
+        description: >
+          Used when switching into Monitor mode.
+          
+          When stress monitoring is enabled, it will start up in stressed mode 
```

Comment:
@kentquirk Is "stress monitoring" better phrased as "Stress Relief" here? I don't see "stress monitoring" mentioned anywhere else, though it is the implied function of the Stress Relief feature.